admin/vagrant.py

```diff
@@ -13,10 +15,25 @@
 
 import flocker
 
-from admin.runner import run
 from admin.release import make_rpm_version
 
 
+def run(command, **kwargs):
```

Comment:
Actually, I guess this outputs the command being run. But, this is just code that got moved, so out of scope to change it and all the existing usage.